src/sql/services/notebook/notebookService.ts

```diff
@@ -20,7 +20,7 @@ export const SERVICE_ID = 'notebookService';
 export const INotebookService = createDecorator<INotebookService>(SERVICE_ID);
 
 export const DEFAULT_NOTEBOOK_PROVIDER = 'builtin';
-export const DEFAULT_NOTEBOOK_FILETYPE = 'IPYNB';
+export const DEFAULT_NOTEBOOK_FILETYPE = 'ipynb';
```

Comment:
Where else are we using / defining `IPYNB`? Are we doing conversion / case-insensitive comparison somewhere? I know our extension passes this in as a capitalized `IPYNB` so am concerned about the impact.

Reply:
I can convert it to lower case and use it.